nimsuggest/nimsuggest.nim

```diff
@@ -590,6 +594,23 @@ proc mainThread(graph: ModuleGraph) =
       conf.suggestionResultHook = proc (s: Suggest) = discard
       recompileFullProject(graph)
 
+when defined(posix):
```

Comment:
Should be moved into its own helper module.

Reply:
Moved to module procmonitor.nim.